docker-images/pysyft-worker/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# Better to not run the worker server without ID
+if [ -z "${WORKER_ID}" ]; then
```

Comment:
Rather than having this as a shell script can we just set a default and do this in the docker file?

Reply:
Doing this in the Dockerfile will require rebuilding the image if we need to use another id, however, being able to provide the id at runtime is better.
What also led me to oblige the user to specify the id is that if we use a default one then the user may be experiencing some issues due to id duplications without knowing what's really happening.

Reply:
+1